frontend/public/components/_edit-yaml.scss

```diff
@@ -36,3 +36,22 @@
 .yaml-editor .monaco-editor .margin .current-line {
   background-color: var(--pf-global--BackgroundColor--dark-200);
 }
+
+.yaml-editor__shortcuts {
+  display: inline-block;
+  margin-right: 10px;
+}
+
+.yaml-editor__icons {
+  position: absolute;
+  top: 5px;
+  right: 0;
+  z-index: 1;
+}
+
+.yaml-editor__icons-divider {
+  color: $color-pf-blue-400;
```

Comment:
Why blue?  The divider isn't clickable.

Reply:
It was blue in the mock. @bmignano was that intentional?

Reply:
It was blue in the mock. @bmignano, was that intentional?

Reply:
The vertical divider line should not be blue, sorry about that! It was a quick mock up :)

Reply:
Thank you! De-blueified:

<img width="1118" alt="Screen Shot 2019-11-01 at 9 18 40 AM" src="https://user-images.githubusercontent.com/7014965/68027396-a07fe980-fc88-11e9-8d20-89798c3f9317.png">